src/js/engagement_view/src/components/reusableComponents/drawer.tsx

```diff
@@ -56,40 +72,55 @@ export const NavigationDrawer = () => {
             </DrawerHeader>
 
             <List className={classes.drawer}>
```

Comment:
More proof of repeated code-smell - this entire section is repeated across files. Maybe this can be extracted out to its own component?

Reply:
Also resolved with react router update!